frontend/packages/kubevirt-plugin/src/components/modals/descheduler-modal/descheduler-modal.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { Alert, Checkbox } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import {
+  createModalLauncher,
+  ModalBody,
+  ModalComponentProps,
+  ModalTitle,
+} from '@console/internal/components/factory';
+import './descheduler-modal.scss';
+import { ModalFooter } from '../modal/modal-footer';
+
+type DeschedulerModalProps = {
+  isVMdeschedulerOn: boolean;
+} & ModalComponentProps;
+
+const DeschedulerModal: React.FC<DeschedulerModalProps> = ({ isVMdeschedulerOn, close }) => {
+  const { t } = useTranslation();
+  const [isOn, setOn] = React.useState(isVMdeschedulerOn); // the default is OFF, the admin has to opt-in this feature
```

Comment:
please add type to `React.useState(isVMdeschedulerOn)` like this `React.useState<boolean>(isVMdeschedulerOn)`